pyproject.toml

```diff
@@ -5,6 +5,24 @@ build-backend = "setuptools.build_meta"
 [project]
 name = "crypt_r"
 version = "3.12.3"
+maintainers = [
+    {name = "Miro Hrončok", email = "miro@hroncok.cz"},
+    {name = "Petr Viktorin", email = "encukou@gmail.com"},
+    {name = "Tomáš Hrnčiar", email = "tomas.hrnciar@me.com"},
+    {name = "Karolina Surma", email = "ksurma@redhat.com"},
+]
+description = "A copy of the `crypt` module that was removed in Python 3.13"
+readme = "README.rst"
+# requires-python = ">=?.??" TODO
```

Comment:
I am not sure which Python version to put here.

Reply:
3.11. On older Pythons, crypt was not deprecated, so I made that arbitrary choice when testing compatibility (see e.g. tox or CI config).